### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Added authentication (Login, signup).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+        closeOnClick
+        rtl={false}
+        draggable
+        pauseOnHover
+      />
       <Navbar />
       <Routes>
-        <Route path='/' element={<LandingPage />}/>
-        <Route path='/home' element={<Home />}/>
+        <Route path='/' element={<LandingPage />} />
```
Comment (sakshi006) [inline src/App.js:26]: You can add an error 404 page. Can refer [this](https://ui.dev/react-router-handling-404-pages)

File: src/Pages/auth/auth.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  padding: 0.5rem 0;
+}
+.input-container {
+  margin: 0;
+  margin-bottom: 1rem;
+}
+.input-section {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #d0d0d0;
```
Comment (sakshi006) [inline src/Pages/auth/auth.css:32]: use CSS variable for assigning color here as well.

**Review by sakshi006 — APPROVED** [review]
Made few comments, looks good to merge
Liked the code structure and naming conventions

File: src/App.js (hunk 1 above)
Comment (SHIKHASINGH1506) [reply src/App.js:26]: Will surely explore this. Thanks for the link though.

File: src/Pages/auth/auth.css (hunk 2 above)
Comment (bhavyaj12) [inline src/Pages/auth/auth.css:32]: Try using color variables

**Review by bhavyaj12 — APPROVED** [review]
Great work. Excellent use of all concepts. Code is clean and well structured.

Can't find a deploy link or screenshots.
Try to add screenshots or gif to pull requests

**Pull request merged**